stanford-earth/matson
# EARTH-1450: Wrapped search js in on window load function.

# NOT READY

# Summary
- wrap search js in a function to load after window load

# Needed By (Date)
- asap

# Steps to Test

1. compile
2. Then test on an ide
3. test all search functionality with opening/closing, adding classes, 
4. search styles should still look correct
5. and that the old search query is removed

# Affected versions or modules
- Does this PR impact any particular module, version, or pull request?

# Associated Issues and/or People
- JIRA ticket SE3:- EARTH-1450 and today's bug fixing
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
